Library/Homebrew/env_config.rb

```diff
@@ -289,6 +289,10 @@ module EnvConfig
                      "outdated.",
         boolean:     true,
       },
+      HOMEBREW_NO_UPDATE_REPORT_ONLY_INSTALLED:  {
```

Comment:
```suggestion
      HOMEBREW_UPDATE_REPORT_ALL_FORMULAE:  {
```
We no longer have a `HOMEBREW_UPDATE_REPORT_ONLY_INSTALLED`, so there's no need to follow the naming convention suggested by it.